content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -363,6 +363,13 @@ relative to their scheduling requests are killed first.
 Unlike pod eviction, if a container is OOM killed, the kubelet can restart it
 based on its `restartPolicy`.
 
+{{<note>}}
+If using [cgroup v2](/docs/concepts/architecture/cgroups/), the kubelet sets the `memory.oom.group` flag so that if a single process in
+a container is OOM killed, all processes in the container will be killed as a group. You can set
+the `singleProcessOOMKill` flag in the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file/) to
+cause kubelet to retain the previous behavior when using cgroup v2.
```

Comment:
```suggestion
cause kubelet to select the legacy kubelet behavior when using cgroup v2.
```
“retain” implies that the cluster was deployed with an older version and then upgraded, which we can't assume.

Reply:
SGTM